src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -1581,8 +1581,11 @@ public class MainActivity : Activity
 				}
 			});
 			using (var b = CreateApkBuilder ("temp/CheckLintErrorsAndWarnings", cleanupOnDispose: false)) {
+				int maxApiLevel = b.GetMaxInstalledPlatform ();
 				string apiLevel;
 				proj.TargetFrameworkVersion = b.LatestTargetFrameworkVersion (out apiLevel);
+				if (int.TryParse (apiLevel, out int a) && a < maxApiLevel)
+					proj.SetProperty ("AndroidLintDisabledIssues", "StaticFieldLeak,ObsoleteSdkInt,AllowBackup,OldTargetApi");
```

Comment:
Maybe the first part of this list, `"StaticFieldLeak,ObsoleteSdkInt,AllowBackup"`, could go in a variable so the list isn't hardcoded twice. Then you could just append: `+ ",OldTargetApi"`

Reply:
done